File: Spring/Chapter7/MyBatisSpringJavaInit/src/main/java/pers/oneToOne/dao/PersonDao.java
package pers.oneToOne.dao;

import pers.oneToOne.entity.Person;
import pers.oneToOne.pojo.PersonPOJO;

public interface PersonDao {
    Person selectPersonById1(Integer id);
    Person selectPersonById2(Integer id);
    PersonPOJO selectPersonById3(Integer id);
}

File: Spring/Chapter7/MyBatisSpringKotlinInit/src/main/kotlin/pers/oneToOne/dao/PersonDao.kt
package pers.oneToOne.dao

import pers.oneToOne.entity.Person
import pers.oneToOne.pojo.PersonPOJO

interface PersonDao {
    fun selectPersonById1(id:Int):Person
    fun selectPersonById2(id:Int):Person
    fun selectPersonById3(id:Int):PersonPOJO
}